vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
Yes.. syncdb didn't work until I fixed settings.py as part of
installation.  Based on your reply, we should keep this change.

Also, there were many steps where we got stuck at installation. I thought
someone fixed the documentation that day. If not I will contribute to that
as well.

Have to take a break now, will post the documentation change later in the
evening.

Also is there a repo of the expectations of the VMS web app  ? I.e. how do
we want to store volunteers, organizations, design of the page, what kind
of data each page should consume etc. ?


On Mon, Oct 19, 2015 at 7:19 AM, Tapasweni Pathak <notifications@github.com>
wrote:

> In vms/vms/settings.py
> <https://github.com/systers/vms/pull/155#discussion_r42363908>:
>
> > @@ -61,8 +61,8 @@
> >      'default': {
> >          'ENGINE': 'django.db.backends.postgresql_psycopg2',
> >          'NAME': 'vms',
> > -        'USER': 'myuser',
> > -        'PASSWORD': 'mypassword',
> > +        'USER': 'vmsadmin',
> > +        'PASSWORD': '0xdeadbeef',
> >          'HOST': 'localhost',
>
> If its intentional to leave it that way, then its fine.
>
> Cool. I thought what you did, does affect the syncdb issue, somehow that
> I'm not aware of. That deadbeef made me curious. [image: :smile:]
>
> Btw, I attended the Codeathon @ Grace Hopper 2015 last week.
>
> That's great. Your first PR does fixes something great.
>
> Still not
> familiar with the standards followed by this project.
>
> No issues. You will learn as you contribute more. Feel free to open issues
> if you don't find something in documentation. If a beginner is not easily
> able to do it, there is some fault in our documentation. You can also fix
> such issues you find
>
—
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/vms/pull/155/files#r42363908>.
>